ePBFT/src/pbft/libbyz/Replica.h

```diff
@@ -152,6 +152,13 @@ class Replica : public Node, public IMessageReceiveBase
   // Effects: Use when messages are passed to Replica rather than replica
   // polling
 
+  bool compare_execution_results(const ByzInfo& info, Pre_prepare* pre_prepare);
```

Comment:
Can you please add a short comments saying these functions are responsible for ledger replay